pkgs/development/python-modules/colout/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, babel
+, buildPythonPackage
+, fetchFromGitHub
+, pygments
+, python3Packages
+, setuptools-scm
+}:
+
+buildPythonPackage rec {
+  pname = "colout";
+  version = "0.12.0";
+
+  src = fetchFromGitHub {
+    owner = "nojhan";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-5ETKNo3KfncnnLTClA6BnQA7SN5KwwsLdQoozI9li7I=";
+  };
+
+  nativeBuildInputs = [
+    babel
+    pygments
+    setuptools-scm
+  ];
+
+  SETUPTOOLS_SCM_PRETEND_VERSION = version;
+
+  propagatedBuildInputs = [
+    babel
+    pygments
+  ];
+
```

Comment:
can we have a `pythonImportsCheck = [ "colout" ];` (I suppose this is how we import it)?